domain/domain.go

```diff
@@ -67,7 +67,7 @@ type Domain struct {
 // It returns the latest schema version, the changed table IDs, whether it's a full load and an error.
 func (do *Domain) loadInfoSchema(handle *infoschema.Handle, usedSchemaVersion int64, startTS uint64) (int64, []int64, bool, error) {
 	var fullLoad bool
-	snapshot, err := do.store.GetSnapshot(kv.NewVersion(startTS))
+	snapshot, err := do.store.GetSnapshotWithPriority(kv.NewVersion(startTS), kv.PriorityHigh)
```

Comment:
All the kv operations for meta data will use high-priority. So we do not need to set it here.

Reply:
Already remove